tuckbox.js

```diff
@@ -458,7 +466,7 @@ function makeBox(
     images
 ) {
   images = images || {};
-  paper = paper === 'a4' ? 'a4' : 'letter';
+  //paper = paper === 'a4' ? 'a4' : 'letter';
```

Comment:
I can't tell what this line is actually doing, so commented it out rather than trying to amend it.